Mathlib/Order/Atoms/Finite.lean

```diff
@@ -28,21 +28,25 @@ variable [LE α] [BoundedOrder α] [IsSimpleOrder α]
 
 section DecidableEq
 
-/- It is important that `IsSimpleOrder` is the last type-class argument of this instance,
-so that type-class inference fails quickly if it doesn't apply. -/
-instance (priority := 200) [DecidableEq α] : Fintype α :=
+/-- It is important that `IsSimpleOrder` is the last type-class argument of this instance,
+so that type-class inference fails quickly if it doesn't apply.
+
+Note that as of 2025-08-13, this is false. Could someone investigate? -/
+scoped instance (priority := 200) [DecidableEq α] : Fintype α :=
```

Comment:
I would guess this creates instance diamonds for \alpha = Bool or `OrderDual Bool` anyway, so indeed is not a good global instance